packages/back-end/src/integrations/Presto.ts

```diff
@@ -51,10 +52,27 @@ export default class Presto extends SqlIntegration {
         user: this.params.username || "",
         password: this.params.password || "",
       };
+      configOptions.user = this.params.username || "";
     }
     if (this.params?.authType === "customAuth") {
       configOptions.custom_auth = this.params.customAuth || "";
```

Comment:
It sounds like on Trino, we were previously not sending a `user` field at all and now we're setting it to `growthbook`.  We're patching that behavior for `basicAuth`.  Do we also need to do something similar for `customAuth`?